packages/docs/src/styles/components/_ReactPropDocs.scss

```diff
@@ -0,0 +1,50 @@
+// Responsive docs table
+
+.docs_table--wrapper {
+  overflow-x: auto;
+}
+
+.docs_table {
+  @media (max-width: $width-sm) {
+    thead {
+      /* Hide table headers (.ds-u-visibility--screen-reader) */
+      border: 0;
+      clip: rect(0, 0, 0, 0);
+      height: 1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+      // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1241631
+      word-wrap: normal;
```

Comment:
I think the only styles you need for `thead` to hide would be 
```
height: 0;
overflow: hidden;
position: absolute;
```

and the others could be removed here

Reply:
Presently, it's coded like the CMS DS Hide table headers (.ds-u-visibility--screen-reader) for accessibility. The suggested change works visually, will check screen-reader now.